### PR TITLE
Ying/changes

### PR DESCRIPTION
**Pull request by brightredchilli**

### CODE REVIEW

File: SwiftWisdom/Core/UIKit/UICollectionView+FlowLayout.swift [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+//
+//
+
+import Foundation
+
+public extension UICollectionView {
+    var flowLayout: UICollectionViewFlowLayout? {
+        return self.collectionViewLayout as? UICollectionViewFlowLayout
+    }
+}
```
Comment (Benuuu) [inline SwiftWisdom/Core/UIKit/UICollectionView+FlowLayout.swift]: newline

**Comment by Benuuu** [comment]
:shipit:

**Comment by loganwright** [comment]
Looks good to me as well after Ben's comment. :shipit:

**Comment by loganwright** [comment]
Ying, can you also update the podspec and tag it? Do you have a trunk account? If you send me your email, I'll add you to auth list for pushing.

**Comment by brightredchilli** [comment]
i will do so. it also appears that...we have a code convention of namespacing extensions...do you guys think I should do that here?

**Pull request merged**